imap_processing/hi/l1a/housekeeping.py

```diff
@@ -0,0 +1,79 @@
+"""Unpack IMAP-Hi housekeeping data."""
+import collections
+import dataclasses
+
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import calc_start_time
+from imap_processing.hi.hi_cdf_attrs import (
+    hi_hk_l1a_attrs,
+    hi_hk_l1a_metadata_attrs,
+)
+
+
+def create_dataset(packets):
```

Comment:
Sorry if this has been discussed already, but how do we feel about type hints? I find them useful if for no other reason than when I step away from IMAP for awhile I need to remind myself what the main objects are that we're all using

```suggestion
from typing import List
from space_packet_parser.parser import Packet

def create_dataset(packets: List[Packet]) -> xr.Dataset:
```

Reply:
I believed it's not been discussed. Cool. I will make that change.